index.js

```diff
@@ -252,6 +254,7 @@ async.map(serviceProviders, function (serviceProviderObj, serviceProviderDone) {
 	});
 }, function (err, results) {
 	// console.log(JSON.stringify(collection, null, 2));
+	outputHandler.close()
```

Comment:
This was a bug from before - without explicitly closing, then file was not actually written.